numba/core/pythonapi.py

```diff
@@ -945,12 +945,14 @@ def call_method(self, callee, method, objargs=()):
 
     def call(self, callee, args=None, kws=None):
         if args is None:
-            args = self.get_null_object()
+            args = self.tuple_new(0)
         if kws is None:
             kws = self.get_null_object()
         fnty = ir.FunctionType(self.pyobj, [self.pyobj] * 3)
         fn = self._get_function(fnty, name="PyObject_Call")
-        return self.builder.call(fn, (callee, args, kws))
+        result = self.builder.call(fn, (callee, args, kws))
+        self.decref(args)
```

Comment:
(Reposting since my previous comment seemed to get lost)

I'm guessing here, but perhaps we should only do this decref if we had to set `args` to a new empty tuple?